spec/services/catalog/soft_delete_spec.rb

```diff
@@ -0,0 +1,16 @@
+describe Catalog::SoftDelete do
+  let(:tenant) { create(:tenant) }
+  let(:portfolio_item) { create(:portfolio_item, :tenant_id => tenant.id) }
+
+  context "when soft-deleting a record" do
```

Comment:
Same here, both contexts (and probably the `let`s on lines 2/3), should be in a `describe ".process" do` block